crispy_forms/templates/bootstrap/layout/radioselect.html

```diff
@@ -5,7 +5,7 @@
     {% include 'bootstrap/layout/field_errors_block.html' %}
 
     {% for choice in field.field.choices %}
-        <label class="radio{% if inline_class %} {{ inline_class }}{% endif %}">
+        <label for="id_{{ field.html_name }}_{{ forloop.counter }}" class="radio{% if inline_class %} {{ inline_class }}{% endif %}">
```

Comment:
I'm happy to merge this then. Don't see any reasons not to.

Can you resolve the conflicts?